game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/food/Foods.kt

```diff
@@ -55,6 +55,11 @@ object Foods {
             p.timers[POTION_DELAY] = 3
         }
 
+        if (food == Food.CAVEFISH) {
+            //  NOT CURRENTLY HANDLED
```

Comment:
in the future please implement content before pushing a pull request. makes reviewing it a lot easier.